### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by Max-0729** [JiaweiShiCV/Amend-Representation-Module#11]
why I used your pretrained model without training that  testing result cannot get close to acc_90 only acc_10??

**Comment by JiaweiShiCV** [comment]
The model accuracy is tested with a batch size of 256. When testing a single image, the lack of a global average representation (GAR) of the batch results in performance degradation.

**Comment by JiaweiShiCV** [comment]
I just updated the code and pretrained model.
Test by running:

```
 python src/test_raf-db.py --checkpoint models/RAF-DB/epoch59_acc0.9205.pth
``` 

Add "-p" or "--plot_cm" if you want to draw a confusion matrix.

**Comment by Max-0729** [comment]
thanks,I will try it.

**Comment by Max-0729** [comment]
thanks, I will try it.

**Comment by Max-0729** [comment]
thanks,I will try it

Jiawei Shi ***@***.***> 於 2021年6月25日 週五 下午1:44寫道：
